### DISCUSSION TITLE
哈密頓力學真偉大！

### DISCUSSION THREAD

**Discussion by LeoTsai999** [1101-GS4719-C/Python-Community-GS4719-C#2143] [General]
哈密頓建立了Hamitonian Mechanics，完善了分析力學的體系並連結巨觀和微觀的力學系統。我覺得這是一個非常重要的貢獻。
這件事啟發了我對於理論研究的想法。我認為一個物理學家，應該要像哈密頓一樣從理論中預測某些目前實驗上還未證實的事情，給予科學研究一個方向！

**Comment by zhilin93517** [comment]
我也有聽過哈密頓力學！以前只要能算出一題哈密頓力學的題目我都會大聲地說：「哈哈！」。
我認為他是一個非常有意思的觀點可以去解析一個力學系統，除了量子力學以外，也可以應用在古典力學中

**Reply by LeoTsai999** [reply]
你也喜歡哈密頓力學嗎？我覺得要不要我們去雙主修物理系？我認為學習哈密頓力學，對我們的未來非常有幫助！

**Reply by zhilin93517** [reply]
我也有興趣修物理系的課，讓物理系的教授帶領我們一起領悟物理世界的美好，霧裡悟理。我認為我們可以去修一門課看看。
